sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/samples/Generated/Samples/Sample_SubscriptionResourceExtensions.cs

```diff
@@ -193,7 +193,7 @@ public async Task CheckDomainAvailability_CheckSKUAvailability()
         public async Task CalculateModelCapacity_CalculateModelCapacity()
         {
             // Generated from example definition: specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2024-10-01/examples/CalculateModelCapacity.json
-            // this example is just showing the usage of "CalculateModelCapacity" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "calculateModelCapacity" operation, for the dependent resources, they will have to be created separately.
```

Comment:
operation id is actually first letter lower case: https://github.com/Azure/azure-rest-api-specs/blob/95aa73b8831deb2940ac7067b8f296cd189a5fe1/specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2024-10-01/cognitiveservices.json#L1080